tests/test_synthetic_pii.py

```diff
@@ -361,3 +362,42 @@ def test_zipcode_mapping():
         expected_proportion_90706,
         rtol=0.1,
     )
+
+
+def test_city_address(mocker):
```

Comment:
Great seeing a unit test!

I do wonder if we need to start building out integration tests for post-processing. It's a long time to wait for tests to pass, though - maybe at the point of post-processing the testing is just done by v&v? @rmudambi?